custom_module/library/kcl_module.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
```

Comment:
Could you move this PR to https://github.com/kcl-lang/ansible-kcl . Thank you!

Reply:
https://github.com/kcl-lang/ansible-kcl/pull/1 

I raised the PR there.